packages/block-editor/src/components/block-list/block.native.js

```diff
@@ -134,7 +141,13 @@ class BlockListBlock extends Component {
 						<BlockInvalidWarning blockTitle={ title } icon={ icon } />
 						}
 					</View>
-					{ isSelected && <BlockMobileToolbar clientId={ clientId } /> }
+					{ isSelected && (
+						<BlockMobileToolbar
+							clientId={ clientId }
+							showSettings={ this.state.showSettings }
+							onSettingsToggle={ this.onSettingsToggle }
+						/>
+					) }
```

Comment:
pass to `showSettings` on `onSettingsToggle` to `BlockMobileTolbar` to allow open/hide `BottomSheet` with additional controls to adjust block settings